man/ocitools-validate.1.md

```diff
@@ -18,6 +21,9 @@ Validate an OCI bundle
 **--path=PATH
    Path to bundle
 
+**--hooks-check**
```

Comment:
On Sun, May 15, 2016 at 10:38:52PM -0700, Ma Shimiao wrote:
> We are not sure if there will be other options to validate going to
> enable/disable particular checks, as hooks is a lit sepcial.  By
> now, there is only hooks need to enablle/disable a check and I think
> an obvious option name is better.

Falling into this same host-path category is root.path (since
opencontainers/runtime-spec#394, currently unreleased).  So maybe the
option should be --host-paths?  Bind mounts and such can also
reference out-of-bundle paths.

I'm sure there are several features like that.  Looking for other
classes of host-specific checks, there's at least “Does the specified
platform match the local system?”.